src/surface/Surface.js

```diff
@@ -207,7 +206,7 @@ class Surface extends Disposable {
 			to = this.defaultChild;
 		}
 		this.activeChild = to;
-		return this.transition(from, to).thenAlways(() => {
+		return this.transition(from, to).then(() => {
```

Comment:
Is this another one that needs a `catch()` to match the old behavior?

Reply:
Yes, correct. Thanks